examples/tutorials/atomic_contributions_tutorial_data/logBB_test_.sdf

```diff
@@ -0,0 +1,2216 @@
+MolID_1

```

Comment:
Same here; can we replace with a smaller file with only one or a few molecules?

Reply:
> 
> 
> Same here; can we replace with a smaller file with only one or a few molecules?

this one can be removed, i can report in tutorial trainset accuracy, if that's fine